soco/core.py

```diff
@@ -1490,6 +1494,70 @@ def get_item_album_art_uri(self, item):
         else:
             return None
 
+    @only_on_master
+    def set_sleep_timer(self, sleep_time_seconds):
+        """Sets the sleep timer.
+
+        Args:
+            sleep_time_seconds (`int/NoneType`): How long to wait before
```

Comment:
You don't need to mark the types as code with ``. Sphinx and Napoleon will take care of that.